sum.py

```diff
@@ -20,7 +20,8 @@
 
 #print the minimum keys
 print("".join(minimum_keys))
-
+print("".join(minimum_keys))
```

Comment:
hey mistakenly you write print statement two times 
please remove that logs. 
thanks.

Reply:
removed print statement thanks for the review